src/chrome/chromeConnection.ts

```diff
@@ -192,4 +191,22 @@ export class ChromeConnection implements IObservableEvents<IStepStartedEventsEmi
                 return (version) ? version : new TargetVersions(Version.unknownVersion(), Version.unknownVersion());
             });
     }
+
+    /**
+     * Gets the port on which chrome was launched, and throw error if the port is not open or accepting connections
+     * @param host The host address on which to check if the port is listening
+     * @param userDataDir Chrome user data directory in which to check for a port file
+     */
+    private async getLaunchedPort(host: string, userDataDir: string): Promise<number> {
+        logger.verbose('Looking for DevToolsActivePort file...');
+        const launchedPort = await chromeUtils.getLaunchedPort(userDataDir);
+        logger.verbose('Got the port, checking if its ready...');
+        const portInUse = await chromeUtils.isPortInUse(launchedPort, host, 100);
+        if(!portInUse) {
+            // bail, the port isn't open
+            logger.verbose('Port not open yet...');
+            throw new Error('Port not open yet');
```

Comment:
We should localize the message, and mention which port we were trying to check

Reply:
Done